content/hi/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -0,0 +1,253 @@
+
+* इस पृष्ठ पर दिखाया गया उदाहरण `kubectl` 1.14 और उससे ऊपर के संस्करण के साथ काम करता है।
+* समझें [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
+
+
+
+<!-- lessoncontent -->
+
+
+## वास्तविक विश्व उदाहरण: कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करना
+
+कॉन्फिगरेशन मैप में स्टोर्ड डेटा का उपयोग करके रेडिस कैश को कॉन्फ़िगर करने के लिए नीचे दिए गए चरणों का पालन करें।
+
+सबसे पहले एक खाली कॉन्फ़िगरेशन ब्लॉक के साथ एक कॉन्फिग मैप बनाएं:
+
+```shell
+cat <<EOF >./example-redis-config.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-redis-config
+data:
+  redis-config: ""
+EOF
+```
+
+रेडिस पॉड मेनिफेस्ट के साथ ऊपर बनाए गए कॉन्फ़िगमैप को लागू करें:
+
+```shell
+kubectl apply -f example-redis-config.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/pods/config/redis-pod.yaml
+```
+
+रेडिस पॉड मेनिफेस्ट की कंटेंट की जांच करें और निम्नलिखित पर ध्यान दें:
+
+* `config` नामक वॉल्यूम `spec.volumes[1]` द्वारा बनाया गया है
+* `spec.volumes[1].configMap.items[0]` के अंतर्गत `कुंजी` और `पथ` `redis-config` कुंजी को उजागर करता है
+  `example-redis-config` `config` वॉल्यूम पर `redis.conf` नामक फ़ाइल के रूप में कॉन्फिग मैप।
+* `config` वॉल्यूम को `spec.containers[0].volumeMounts[1]` द्वारा `/redis-master` पर माउंट किया जाता है।
+
+इसका `example-redis-config` से `data.redis-config` में डेटा को उजागर करने का शुद्ध प्रभाव है
+पॉड के अंदर `/redis-master/redis.conf` के रूप में ऊपर कॉन्फिग मैप।
+
+{{% code_sample file="pods/config/redis-pod.yaml" %}}
+
+बनाई गई वस्तुओं की जांच करें:
+
+```shell
+kubectl get pod/redis configmap/example-redis-config 
+```
+
+आपको निम्नलिखित आउटपुट देखना चाहिए
+
+```
+NAME        READY   STATUS    RESTARTS   AGE
+pod/redis   1/1     Running   0          8s
+
+NAME                             DATA   AGE
+configmap/example-redis-config   1      14s
+```
+
+याद रखें कि हमने `example-redis-config` कॉन्फिग मैप में `redis-config` कुंजी को खाली छोड़ दिया था:
+
+```shell
+kubectl describe configmap/example-redis-config
+```
+
+आपको एक खाली `redis-config` कुंजी दिखनी चाहिए:
+
+```shell
+Name:         example-redis-config
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+redis-config:
+```
+
+पॉड में प्रवेश करने के लिए `kubectl exec` का उपयोग करें और वर्तमान कॉन्फ़िगरेशन की जांच करने के लिए `redis-cli` टूल चलाएं:
+
+```shell
+kubectl exec -it redis -- redis-cli
+```
+
+चेक `maxmemory`:
+
+```shell
+127.0.0.1:6379> CONFIG GET maxmemory
+```
+
+इसे 0 का डिफ़ॉल्ट वैल्यू दिखाना चाहिए:
+
+```shell
+1) "maxmemory"
+2) "0"
+```
+
+इसी तरह, `maxmemory-policy` जांचें:
+
+```shell
+127.0.0.1:6379> CONFIG GET maxmemory-policy
+```
+
+जिससे इसका डिफ़ॉल्ट वैल्यू भी प्राप्त होना चाहिए`noeviction`:
+
+```shell
+1) "maxmemory-policy"
+2) "noeviction"
+```
+
+आइए अब `example-redis-config` कॉन्फिग मैप में कुछ कॉन्फ़िगरेशन वैल्यू जोड़ें:
+
+{{% code_sample file="pods/config/example-redis-config.yaml" %}}
```

Comment:
The same build error seems to apply to the `example-redis-config.yaml` file. I recommend following a similar approach by copying the [en/docs/tutorials/configuration/pods/config/example-redis-config.yaml](https://github.com/kubernetes/website/blob/main/content/en/examples/pods/config/example-redis-config.yaml) file to **`hi/docs/tutorials/configuration/pods/config/example-redis-config.yaml.`** This could help address the error with this file.

Reply:
Thanks, @dipesh-rawat, but no example folder is in the ( content/hi ). So first of all we need to create an example folder under content/hi to create yaml file.
![git](https://github.com/kubernetes/website/assets/124805747/3663d89d-2cc2-456d-94a9-780910a4dc2f)

Reply:
Refer https://github.com/kubernetes/website/pull/44794#discussion_r1461258684